sub-packages/bionemo-moco/scripts/clean_documentation.py

```diff
@@ -0,0 +1,43 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Isn't there a way to launch/view docs built/rebuilt by our regular documentation server? https://github.com/NVIDIA/bionemo-framework/tree/main/docs for example has a readme on this.

Reply:
I'm worried about something like this being the implied standard for displaying/rendering docs for users who want to explore and test things. Better to make the actual way of doing it easier than hack a new way IMO, unless there is a good reason.

Reply:
Per our discussion can you add a README to this directory describing what these scripts are, how to run them, and what to use them for? Thanks!

Reply:
Done